touchforms/backend/xformplayer.py

```diff
@@ -445,7 +445,7 @@ def _parse_repeat_juncture(self, event):
         ro = r.getRepeatOptions()
 
         event.update(get_caption(r))
-        event['main-header'] = ro.header
```

Comment:
@benrudolph Can you describe in greater detail what's happening here and where that name change will show up?  I did a grep for `main-header` and I didn't see anywhere in the smsforms code that is looking for it so we're probably ok, but would be good to know in greater detail where this might show up.